src/metabase/api/dataset.clj

```diff
@@ -69,7 +69,7 @@
         (qp-runner query info context)))))
 
 (api/defendpoint ^:streaming POST "/"
-  "Execute a query and retrieve the results in the usual format."
+  "Execute a query and retrieve the results in the usual format. The query will not use the cache."
```

Comment:
@paoliniluis Let me know if this covers what you were thinking about in https://github.com/metabase/metabase/issues/25071.

I didn't update the description for `/api/card/:id/query` since it sounded like it was specific to a parameter (so there's an explicit signal that the cache isn't used in that case).

Reply:
yup, it's perfectly fine. According to the code, this endpoint will never use the cache :)